tests/installation/setup_online_repos.pm

```diff
@@ -20,8 +20,13 @@ use version_utils;
 
 sub run {
     # ordered according to real repos lists
-    my @default_repos = qw(update-non-oss update-oss main-non-oss main-oss debug-main untested-update debug-update source);
-
+    my @default_repos;
```

Comment:
lists are totally identical just having different order 
```suggestion
    my @default_repos = qw(update-non-oss update-oss main-non-oss main-oss debug-main untested-update debug-update source);
```
and you can drop `if\else` thingy below or I missing something ?

Reply:
Yes, list must be in correct order, otherwise it will fail. Order of the repositories is different between 15.0 and 15.1.

Reply:
yeah , I have just read the logic which using this . still looks fishy but this is not related to this PR for sure ...